package.json

```diff
@@ -10,7 +10,12 @@
   },
   "devDependencies": {
     "@automattic/color-studio": "^2.5.0",
-    "@babel/eslint-parser": "7.15.4",
+		"@babel/core": "7.28.3",
+		"@babel/eslint-parser": "^7.23.3",
+		"@babel/plugin-proposal-optional-chaining": "^7.21.0",
```

Comment:
Are we using this anywhere? AFAICT, we replaced it with `@babel/plugin-transform-optional-chaining` from below?